README.md

```diff
@@ -1,20 +1,32 @@
 # cv-datasheets
-
+Datasheets are documents that can be offered to help make datasets more valuable to the developers, researchers or other people who use the dataset. We're working with the [Common Voice](https://commonvoice.mozilla.org/) language communities to support the creation of a [datasheet](https://en.wikipedia.org/wiki/Datasheet) for each of the languages in the Common Voice dataset.
 ## Community contribution
+The Common Voice team believes that language communities are the experts that should should steer the direction of their languages on Common Voice. We're asking members of the hundreds of language communities across Common Voice to help create datasheets for your language(s), because you're going to be the best qualified to describe the features of your language(s).
+
+**Example datasheet:** The datasheet for [Norsk Nynorsk — Norwegian Nynorsk (nn-NO)](https://github.com/common-voice/cv-datasheets/blob/main/cv-corpus/scs/23.0-2025-09-17/final/en/nn-NO.md) can be a helpful guide to what kind of information to include in a new datasheet. Information on the amount of recorded hours for this dataset, validated hours and demographics in this dataset are automatically generated.
+
+As Common Voice Scripted Speech and Spontanous Speech are seperate datasets, seperate datasheets will need to be created for languages that collect data in both of these modes.
+
+Right now, the information needed to generate new datasheets can be submitted via a Google Form or via a Pull Request on this GitHub repository. If you haven't used GitHub before, we recommend following the instructions below to submit a datasheet for your language via Google form.
+
+If you can't access the Google form or the GitHub contribution process, please email the Common Voice team at commonvoice@mozilla.com for support.
+
+
 
 ### Via a form
+To add a new datasheet for your language in either the Scripted Speech or Spontaneous Speech, please click the appropriate link below and follow the instructions in the form.
+
+Currently these forms are available in English and Spanish. To request a version of this form in your language please email commonvoice@mozilla.com with the subject line "Datasheet form localization".
```

Comment:
Maybe we could just give them a link to a google sheet for the localisation, where they can request access. In the interests of reducing emails to a minimum.